tools/src/test/java/org/terracotta/utilities/concurrent/InterprocessCyclicBarrierTest.java

```diff
@@ -55,6 +64,49 @@ public class InterprocessCyclicBarrierTest  {
   @Rule
   public final TemporaryFolder temporaryFolder = new TemporaryFolder();
 
+  @Rule
+  public final Timeout timeout = Timeout.seconds(10L);
+
+  @Rule
+  public final TestWatcher watcher = new TestWatcher() {
```

Comment:
This was added to help identify the hanging test -- under normal conditions, these tests produce no output making a stalled test impossible to identify.